flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/watermarkstatus/WatermarkStatus.java

```diff
@@ -28,11 +28,11 @@
 /**
  * A Stream Status element informs stream tasks whether or not they should continue to expect
```

Comment:
Unfortunately, it seems that we also use `Stream status` in our codebase you probably also need to replace those occurrences.

Reply:
I added 3 more grep patterns. Good catch!